cmd/collector/app/zipkin/jsonv2.go

```diff
@@ -0,0 +1,179 @@
+// Copyright (c) 2017 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package zipkin
+
+import (
+	"github.com/jaegertracing/jaeger/model"
+	"github.com/jaegertracing/jaeger/swagger-gen/models"
+	"github.com/jaegertracing/jaeger/thrift-gen/zipkincore"
+)
+
+func spansV2ToThrift(spans *models.ListOfSpans) ([]*zipkincore.Span, error) {
```

Comment:
is the pointer to the slice required? It doesn't look like you're doing anything to the slice reference in this function. Without the pointer, this function should be more readable